plugins/CoreHome/stylesheets/dataTable/_dataTable.less

```
@@ -329,20 +329,22 @@ table.dataTable img {
 }
 
 .datatableRelatedReports {
-  color: #888;
+  color: #808080;
```

Minor tweak but technically 888 becomes 888888 unless it's to also address contrast issue etc.
```suggestion
  color: #888888;
```

---

That one had been overwritten somewhere else. I just combined the two rules.